frontend/src/AnimalCard/AnimalCard.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import "./AnimalCard.css";
+import exp from "constants";
```

Comment:
What is exp? Do u really need this import?

Reply:
wie didn't need it, thanks